src/assignments/flask-game.rst

```diff
@@ -10,14 +10,16 @@ from start to finish.
 Now that you have some Flask experience, your team can move the game out of the
 console and into a browser!
 
+.. figure:: figures/RPS.png
```

Comment:
I think this figure would be better if there were more of a distinction/division between the right and left sides.  From a student's point of view, I see everything mushed together in the same workspace, so I'm not sure what's supposed to be different.

Can you configure the image to show the console output on the left, then some white space, then a screenshot from a browser on the right?

Also, how about going deep into the RPS Flask game (CSS styling, feedback messages, maintaining the form so the user can play again, etc.)?  Let's show the kids what's possible!

Last, Giphy Capture is a program in the App store.  I used it to record gifs for some of the pther pages, and these can be inserted just like an image.  Maybe a gif would demo the Flask app better.

Reply:
Let me know what you think of the new image